File: Homework/CSharp-Part-1/03. Operators-and-Expressions/PointInACircle.cs
using System;

class PointInACircle
{
    static void Main()
    {
        double x = double.Parse(Console.ReadLine());
        double y = double.Parse(Console.ReadLine());
        double radius = 2;

        double distance = Math.Sqrt(x * x + y * y);

        Console.WriteLine((distance <= radius) ?
                          String.Format("yes {0:0.00}", distance) :
                          String.Format("no {0:0.00}", distance));
    }
}

File: Homework/JavaScript-Fundamentals/04. Operators and Expressions/point-in-a-circle.js
function solve(args) {
    // PointInACircle
    const radius = 2;

    var x = +args[0];
    var y = +args[1];

    var distance = Math.sqrt(x * x + y * y);

    console.log((distance <= radius) ?
        'yes ' + distance.toFixed(2) :
        'no ' + distance.toFixed(2));
}